data/transactions/logic/fields.go

```diff
@@ -195,7 +195,12 @@ var txnFieldSpecs = []txnFieldSpec{
 	{Sender, StackBytes, 0, 5, false},
 	{Fee, StackUint64, 0, 5, false},
 	{FirstValid, StackUint64, 0, 0, false},
-	{FirstValidTime, StackUint64, 0, 0, false},
+
+	// FirstValidTime was originally a dummy field in , and enabled in v6.
```

Comment:
I think the comment should have been:

`FirstValidTime was originally a dummy field in v0-v5, and enabled in v6.`